shed/event_streams.py

```diff
@@ -254,33 +257,31 @@ def curate_streams(self, nds, outbound):
         If we get multiple streams make (name, (doc, doc, doc, ...))
         Otherwise (name, (doc,))
         """
+        # if there are multiple streams
+        if isinstance(nds[0], tuple):
```

Comment:
but  `( ('start', {}), ('start', {}) )  ` is a possible input, so I was looking at the extreme case (a tuple of length 1). I don't think any stream seems to be doing this though.